src/applications/proxy-rewrite/tests/health.integration.spec.js

```diff
@@ -0,0 +1,44 @@
+const puppeteer = require('puppeteer');
+
+describe('jest-image-snapshot usage with an image received from puppeteer', () => {
+  let browser;
+  jest.setTimeout(20000);
+  beforeAll(async () => {
+    browser = await puppeteer.launch();
+  });
+
+  it('compares screenshots of elements', async () => {
+    const page = await browser.newPage();
+    page.setViewport({ width: 1024, height: 800 });
+    await page.goto('https://preview.va.gov/health/');
```

Comment:
This URL should probably be moved to a const that all tests use.

Reply:
Good idea, updated.